Model/Api/Shipping.php

```diff
@@ -86,17 +86,21 @@ public function __construct(
     /**
      * @param array $addressData
      * @param null $shipping_option
+     * @param null $ship_to_store_option
      * @return ShippingDataInterface
      * @throws LocalizedException
      */
-    public function generateResult($addressData, $shipping_option)
+    public function generateResult($addressData, $shipping_option, $ship_to_store_option)
```

Comment:
Looks like we do not use new parameter `$ship_to_store_option` as well as existing parameter `$shipping_option`

Reply:
@vitaliyreznikov we have an abstract function in its parent class ShippingTax, both shipping api class and tax api class extend this abstract class, 
```
abstract public function generateResult($addressData, $shipping_option, $ship_to_store_option); 
```
And for shipping api hook, it only sends addressData in the request, then the tax api hook has shipping_option or ship_to_store_option data.